### PR TITLE
add atorch dev env.

### PR DESCRIPTION
**Pull request by nash635**
add atorch dev env.

### CODE REVIEW

File: atorch/atorch/requirements.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 transformers==4.17
 networkx
 pyomo
 pynvml==11.4.1
-flash-attn>=0.2.1
-grpcio==1.34.1
-grpcio-tools==1.34.1
+# flash-attn>=0.2.1
+# grpcio==1.34.1
+# grpcio-tools==1.34.1
```
Comment (workingloong) [inline atorch/atorch/requirements.txt:11]: It is better to remove the unused lines.

File: atorch/dev/docker/pip.conf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[global]
+# index-url = https://pypi.antfin-inc.com/simple
```
Comment (workingloong) [inline atorch/dev/docker/pip.conf:2]: Remove it.

File: atorch/dev/docker/pip.conf [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+index-url = http://mirrors.aliyun.com/pypi/simple
+[install]
+trusted-host = pypi.antfin-inc.com mirrors.aliyun.com
```
Comment (workingloong) [inline atorch/dev/docker/pip.conf:5]: Remove `pypi.antfin-inc.com`.

File: atorch/dev/docker/Dockerfile-base [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    export eval ${CMAKE_VARS} && \
+    TORCH_CUDA_ARCH_LIST="3.5 5.2 6.0 6.1 7.0+PTX 8.0" TORCH_NVCC_FLAGS="-Xfatbin -compress-all" \
+    CMAKE_PREFIX_PATH="$(dirname $(which conda))/../" \
+    python setup.py install
+
+FROM conda as conda-installs
+ARG PYTHON_VERSION=3.8
+ARG CUDA_VERSION=11.7
+ARG CUDA_CHANNEL=nvidia
+ARG INSTALL_CHANNEL=pytorch-nightly
```
Comment (skydoorkai) [inline atorch/dev/docker/Dockerfile-base:67]: Do not use nightly. Use a certain version.

**Review by workingloong — APPROVED** [review]
LGTM

**Pull request merged**